soccer_ipm:
  ros__parameters:
    balls:
      ball_topic: 'balls_in_image'
      ball_radius: 0.075
      
    goalposts:
      goalposts_topic: 'goal_posts_in_image'
      footpoint_out_of_image_threshold: 0.8
      bar_height: 2.0

    obstacles:
      obstacles_topic: 'obstacles_in_image'
      footpoint_out_of_image_threshold: 0.8

    field_boundary:
      field_boundary_topic: 'field_boundary_in_image'

    camera_info:
      camera_info_topic: 'camera/camera_info'

    masks:
      line_mask:
        topic: 'line_mask_in_image'
        scale: 0.25
